### DISCUSSION TITLE
Any eta for dompdf 1.2.0?

### DISCUSSION THREAD

**Discussion by shyim** [dompdf/dompdf#2704] [General]
Hey,

I am making currently a project ready for PHP 8.1 and saw that the master have many fixes for this PHP version.
Is there any eta when this will be released?

**Comment by fd6130** [comment]
You can directly use the master version, just use vcs in composer.json. But beware, master keep changing mean it will constantly break your application.

**Comment by bsweeney** [comment]
There are some outstanding updates we're trying to work though. Trying to decide if we should go ahead and release now and include those updates in a follow-on release. Either way the next release should be coming soon.

**Reply by shyim** [reply]
Any new updates on this?

**Reply by bsweeney** [reply]
Obviously decided to wait and let some final updates roll in. The release is pretty much ready just making some final updates.

**Comment by bsweeney** [comment]
1.2.0 is now available
